tests/services/email.test.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-unused-expressions */
+import chai from 'chai';
```

Comment:
Can we do 
```suggestion
import { should } from 'chai';
```
and then `should()`?

Reply:
i needed the chai to use chai-as-promised